pkg/quarkus/v1alpha/scaffolds/internal/templates/makefile.go

```diff
@@ -62,6 +69,9 @@ func (f *Makefile) SetTemplateDefaults() error {
 }
 
 const makefileTemplate = `
+IMAGE_TAG_BASE ?= example.com/memcached-quarkus-operator
```

Comment:
This can not be hardcoded to `example.com/memcached-quarkus-operator`. We're doing something in the `manifests` plugin with this. See https://github.com/operator-framework/operator-sdk/blob/master/internal/plugins/manifests/v2/init.go#L148-L153 I think the `%[1]s/%[2]s` is some Go template logic that gets replaced when the plugin is run. We will want to do something similar.

Reply:
I will go through this `Go template logic` once and update this PR.

Reply:
Working on it.

Reply:
Updated this tag and added in the same way defined above.